### ISSUE TITLE
Minimum spec required

### ISSUE THREAD

**Issue by danboid** [HenrykRichter/libSDL12_Amiga68k#2]
What is the minimum spec CPU and RAM required to run SDL under AmigaOS? Please add this to the README.

Would it be possible to run a simple Amiga SDL app on an A600 with 5 MB RAM?

**Comment by MBeijer** [comment]
No, SDL1.2 requires at least a 68040 and AGA. This version of SDL1.2 for Amiga requires RTG.

**Comment by danboid** [comment]
OK, thats higher requirements than I expected. I think this should be added to the README.

**Comment by MBeijer** [comment]
SDL is not created with the Amiga architecture in mind, where the CPU is not the main chip, but rather only there to shuffle data, which is then offloaded by the custom chips. SDL is designed around the PC. The only reason we have SDL for Amiga is to backport PC-games, which usually rely on CPU and GPU.

This repo is also **_not_** maintained, the current repo is here: https://github.com/AmigaPorts/SDL-1.2

The requirements are stated in the Aminet releases: 
* https://aminet.net/package/dev/lib/SDL_RTG
* https://aminet.net/package/dev/lib/SDL_AGA

But should probably be mentioned in the repository mentioned above.
